config/global.ini.php

```diff
@@ -209,6 +209,12 @@
 enabled_periods_UI = "day,week,month,year,range"
 enabled_periods_API = "day,week,month,year,range"
 
+; whether to enable segment archiving cache
+; Note: if you use any plugins, this need to be compliant with Matomo and
+; * depending on the segment you create you may need a newer MySQL version (eg 5.7 or newer)
+; * use a reader database for archiving in case you have configured a database reader
+enable_segments_cache = 1
```

Comment:
we would disable this before the release

Reply:
and we'll need to write tests with this setting enabled and then check the segmentation works